### PR TITLE
Admin System Feedback Changes

### PR DESCRIPTION
**Pull request by Gandalf2k15**
You no longer need to use the system tab at all for you lazy spergs. All you gotta do is click one of those (R) buttons riddled around to resolve your adminhelp. For the love of god resolve them once your done. Its dynamic, so if you PM someone it wont show up, eg intelligent.

Fixes that silly access bug.

image

Logging now functions correctly too.
image

### CODE REVIEW

**Comment by calibraptor** [comment]
With how fast chat tends to scroll this'll only be useful for shitposts that can be resolved instantly.

**Comment by Gandalf2k15** [comment]
Erry thing has its use

**Comment by Spacedong** [comment]
yeah im really down for this +1

**Comment by ghost** [comment]
Greato stuff +1

**Comment by Gandalf2k15** [comment]
Also fixes silly access bug.

File: code/modules/admin/verbs/adminpm.dm [hunk 1]
```diff
@@ -128,20 +128,33 @@
 
 	msg = emoji_parse(msg)
 	var/keywordparsedmsg = keywords_lookup(msg)
+	var/datum/adminticket/ticket
 
 	if(C.holder)
 		if(holder)	//both are admins
 			for(var/datum/adminticket/T in admintickets)
-				if(T.permckey == C.ckey && T.resolved == "No")
+				if(T.permckey == src.ckey && T.resolved == "No")
```
Comment (Carbonhell) [inline code/modules/admin/verbs/adminpm.dm:136]: > src.
> god damn how many times do i have to say this

**Comment by Carbonhell** [comment]
Code's ok,needs only another head ok

**Comment by dreamcrab** [comment]
sure

**Comment by dreamcrab** [comment]
+777777777777777777777

**Pull request merged**